common/src/autogluon/common/utils/s3_utils.py

```diff
@@ -39,6 +39,92 @@ def delete_s3_prefix(bucket, prefix):
         s3.meta.client.delete_objects(Bucket=bucket, Delete=delete_keys)
         
 
+def upload_file(file_name: str, bucket: str, prefix: Optional[str] = None):
+    """
+    Upload a file to a S3 bucket
+
+    Parameters
+    ----------
+    file_name: str,
+        File to upload
+    bucket: str,
+        Bucket to upload to
+    prefix: Optional[str], default = None
+        S3 prefix. If not specified then will upload to the root of the bucket
+    """
+    import boto3
+    from botocore.exceptions import ClientError
+
+    object_name = os.path.basename(file_name)
+    if len(prefix) == 0:
+        prefix = None
+    if prefix is not None:
+        object_name = prefix + "/" + object_name
+
+    # Upload the file
+    s3_client = boto3.client("s3")
+    try:
+        s3_client.upload_file(file_name, bucket, object_name)
+    except ClientError as e:
+        raise e
```

Comment:
try/except seems to do nothing currently

Reply:
Oh yea, should just remove it

Reply:
Still present in latest commit

Reply:
should be fixed now